### DISCUSSION TITLE
2022/01/26/%E5%B0%9D%E8%AF%95%E5%86%99%E6%96%87%E7%AB%A0/

### DISCUSSION THREAD

**Discussion by giscus** [Westlifers/Westlifers.github.io#9] [Announcements]
# 2022/01/26/%E5%B0%9D%E8%AF%95%E5%86%99%E6%96%87%E7%AB%A0/

This is the very first post of mine for this site. I’d test some features of markdown, and practice markdown, of course.Markdown is powerful. Nevertheless I have no idea if hexo supports mathematics f

http://localhost:4000/2022/01/26/%E5%B0%9D%E8%AF%95%E5%86%99%E6%96%87%E7%AB%A0/

**Comment by Westlifers** [comment]
nice post, have to say. Wondering how to render math formula in comments.
$a^2=x^3$ isn't rendered as expected

**Reply by Westlifers** [reply]
no way, man. Giscus doesn't support this...

**Comment by Westlifers** [comment]
Trying to write comments in different place but still refer to Github Discussion. Watch if comments show correctly.
